### PR TITLE
Configurable Rust Analyzer inlay face.

### PR DESCRIPTION
**Pull request by mpereira**
There are some unrelated formatting changes... let me know if you want me to revert them.

### CODE REVIEW

**Comment by brotzeit** [comment]
Thanks for contributing!

File: lsp-rust.el [hunk 1]
```diff
@@ -456,6 +456,12 @@ The command should include `--message=format=json` or similar option."
   :group 'lsp-rust
   :package-version '(lsp-mode . "6.3.2"))
 
+(defcustom lsp-rust-analyzer-inlay-face 'font-lock-comment-face
+  "The face to use for the Rust Analyzer inlays."
+  :type 'face
+  :group 'lsp-rust
+  :package-version '(lsp-mode . "6.3.2"))
```
Comment (yyoncho) [inline lsp-rust.el:463]: can you use 6.4?

Comment (mpereira) [reply lsp-rust.el]: 9ee929e281375765afda05995badb45a3e523347

**Comment by yyoncho** [comment]
Thank you for contributing to `lsp-mode` and congrats for your first `lsp-mode` PR! :tada:

**Pull request merged**